scripts/deploy-package-manifest.sh

```diff
@@ -0,0 +1,47 @@
+set -x
```

Comment:
Could we name this scripts/deploy-catalogsource.sh? Really what we are creating here is the CatalogSource (and OperatorGroup). The Catalogsource is what drives OLM to deploy the packagemanifests for us. The script is the same just a simple rename.